### DISCUSSION TITLE
初學 Python 遇到瓶頸嗎？5 個問題及應對方式讓你更上一層樓！

### DISCUSSION THREAD

**Discussion by ru0372572** [1101-GS4719-C/Python-Community-GS4719-C#752] [Ideas]
學程式不用怕！網路資源與同儕討論相輔相成，幫助我們一起走向長遠的道路。
快來約第一次小組討論吧！！！

https://hahow.in/contents/articles/6255464e131ea20006ccbd87

**Comment by manipulateed** [comment]
這篇文章能幫助還在程式設計大門前徘徊的人，有了更好的說服而勇敢堅持的走下去。

**Comment by dinosaur1009** [comment]
看完之後真的對於程式比較有信心了.....以前會很害怕有錯誤訊息之類的 現在好像可以慢慢適應了

**Comment by Chiway1229** [comment]
當時在接觸二分搜時 真的學習到超崩潰 也考慮過放棄這條路
不過慶幸我有再給自己一個機會 相信這次能不一樣

**Comment by evanchou1204** [comment]
看完這篇對我這個初學者幫助蠻大，讓我釐清自己學python的目的，也讓我能訂定目標，知道要在大量的練習和錯誤中建立正確的觀念。

**Comment by Pietra1226** [comment]
謝謝這篇貼文，有受到很大的幫助，還有幫助我知道要如何培養一些寫程式的好習慣，也讓我更了解python

**Comment by williamyen0623** [comment]
謝謝版主的分享！相當實用的文章，自己學習程式語言常常卡關，算是幫了個大忙

**Comment by swerfg03joe** [comment]
程式語言是適合任何人學習的，會害怕通常是被外界給影響，例如電影都是演某駭客鍵盤隨便敲一敲就有屌炸天的功能，但事實上坐在電腦前抓bug的景象才是真的，不要害怕學習，都能讀大學了程式語言只是個工具而已，照著語言規範再加上自己的邏輯就可以越來越優秀

**Comment by 710388** [comment]
感覺github加csdn能解決多數問題，後者的理論解釋真的很不錯，新手可以多試試

**Comment by OHoln** [comment]
我自己覺得學python可以先找一個自己有興趣的專案主題，再去了解這個主題背後用的的技術和觀念，比較不會像無頭蒼蠅一樣不知道現階段要做甚麼

**Comment by Piyenparty** [comment]
我覺得在學習路上，相互詢問真的是很好的做法，可以節省超多時間，尤其像撰寫程式這種吃大量練習以及經驗的事情，若有一位已經通透相關知識的人可以幫忙，相信學習效率會大幅提升，所以我很贊同你用學習小組的做法，真心支持
